### PR TITLE
Fix/new cache

### PR DESCRIPTION
**Pull request by antoniogamiz**

### CODE REVIEW

File: lib/Documentable/DocPage/Factory.pm6 [hunk 1]
```diff
@@ -1,6 +1,5 @@
 
 use Documentable;
-use Pod::Load;
 use Documentable::Config;
```
Comment (JJ) [inline lib/Documentable/DocPage/Factory.pm6:3]: Did you check if it's no longer used to eliminate it from dependencies?

File: t/104-update-issue-16-47.t [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    lives-ok {
+        Documentable::CLI::MAIN('start', :topdir('t/test-doc'), :dirs('Language'), :p)
+    }, "Lives when a file is deleted";
+
+}
+
+rmtree("t/.cache-test-doc");
+rmtree("html");
+
+done-testing;
```
Comment (JJ) [inline t/104-update-issue-16-47.t:34]: Cool this.

File: lib/Documentable/DocPage/Factory.pm6 (hunk 1 above)
Comment (antoniogamiz) [reply lib/Documentable/DocPage/Factory.pm6:3]: It's still used in the tests.

File: lib/Documentable/CLI.pm6 [hunk 3]
```diff
@@ -333,6 +328,7 @@ package Documentable::CLI {
             :$topdir,
             :dirs(DOCUMENTABLE-DIRS),
             :verbose($verbose),
+            :$typegraph-file
```
[the remark this answers is not in the document]
Comment (antoniogamiz) [reply lib/Documentable/CLI.pm6:331]: Yep, but after fixing the update option that error appeared.

**Pull request merged**